src/scss/bootpresso.scss

```diff
@@ -37,3 +39,7 @@
 @import "bootstrap/scss/_carousel.scss";
 @import "bootstrap/scss/_utilities.scss";
 @import "bootstrap/scss/_print.scss";
+
+// custom components
```

Comment:
what does custom refer to here? is it still a regular bootstrap component with custom styling? or a custom component?